### ISSUE TITLE
CentOS: undefined symbol: _ZN3aws14lambda_runtime7runtimeC1ERKSsS3_

### ISSUE THREAD

**Issue by jkutner** [aws/aws-lambda-nodejs-runtime-interface-client#14]
I'm trying to install the module on CentOS, and the `preinstall.sh` script succeeds, but when I try to run the executable, it gives the error:

```
$ npx aws-lambda-ric index.handler
2021-02-13T16:08:11.247Z        undefined       INFO    Executing 'index.handler' in function directory '/workspace'
node: symbol lookup error: /workspace/node_modules/aws-lambda-ric/build/Release/runtime-client.node: undefined symbol: _ZN3aws14lambda_runtime7runtimeC1ERKSsS3_
```

I think this might be helped or fixed by #8

**Comment by jkutner** [comment]
I tested #8 and it did fix the problem

**Comment by anthony-telljohann** [comment]
I'm also experiencing this error when I try to build a CentOS 7 docker image for my lambda.

**Comment by andclt** [comment]
Hi, this issue has been fixed with the following [PR](https://github.com/aws/aws-lambda-nodejs-runtime-interface-client/pull/57) which is included in the 2.1.0 release https://www.npmjs.com/package/aws-lambda-ric/v/2.1.0
